.github/workflows/ci.yml

```diff
@@ -481,39 +481,51 @@ jobs:
       fail-fast: false
       matrix:
         include:
+          - name: ubu22-x86-gcc12-clang-repl-19-docs
+            os: ubuntu-22.04
+            compiler: gcc-12
+            clang-runtime: '19'
+            cling: Off
+            cppyy: Off
+            docs: On
           - name: ubu22-x86-gcc12-clang-repl-19-cppyy
             os: ubuntu-22.04
             compiler: gcc-12
             clang-runtime: '19'
             cling: Off
             cppyy: On
             coverage: true
+            docs: Off
```

Comment:
I think we `Off` is assumed. Can we remove the boilerplate?

Reply:
I'll do this in the next commit. Just going to see whether the ci builds the documentation first.